src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpTestForEmptyStringsUsingStringLength.Fixer.cs

```diff
@@ -37,5 +37,9 @@ protected override SyntaxNode GetRightOperand(SyntaxNode binaryExpressionSyntax)
         {
             return ((BinaryExpressionSyntax)binaryExpressionSyntax).Right;
         }
+
+        protected override bool ContainsEmptyStringLiteral(SyntaxNode node)
```

Comment:
I think this can be implemented in the base type using [SemanticModel.GetConstantValue](http://sourceroslyn.io/#Microsoft.CodeAnalysis/Compilation/SemanticModel.cs,744) API